packit_service/sentry_integration.py

```diff
@@ -66,6 +83,7 @@ def configure_sentry(
         secret_key,
         integrations=integrations,
         environment=getenv("DEPLOYMENT"),
+        traces_sampler=traces_sampler,
```

Comment:
I've briefly read through the docs, this does not affect the exceptions, right? Because the docs suggest so :D

Reply:
Can you be more specific? What docs, where? I couldn't find it.

Reply:
from the comment https://docs.sentry.io/platforms/python/configuration/sampling/

oh… i see… I got confused cause docs mention sampling of »the errors« as the first thing, but this is just for traces.